### PR TITLE
Release v0.12.2

### PR DESCRIPTION
**Pull request by kozlovic**
Signed-off-by: Ivan Kozlovic <ivan@synadia.com>

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -1,4 +1,4 @@
-FROM golang:1.11.5
+FROM golang:1.11.6
```
Comment (kozlovic) [inline Dockerfile]: Actually, looks like docker image 1.11.6 not yet available. So if doing release today, will have to keep 1.11.5

Comment (kozlovic) [reply Dockerfile]: Will keep checking on avail.. may be coming today?

Comment (derekcollison) [reply Dockerfile]: idk

Comment (kozlovic) [reply Dockerfile]: avail now, so merging.

**Pull request merged**